src/wallet/wallet.cpp

```diff
@@ -842,6 +842,25 @@ bool CWallet::IsSpentKey(const uint256& hash, unsigned int n) const
     return false;
 }
 
+void CWallet::SetUsedAddressState(WalletBatch& batch, const CTxOut& cout, bool used)
```

Comment:
`used` is always `true`, maybe drop it?

Reply:
This has been resolved in the latest commit, thanks!